tests/Tests.AzureAppConfiguration/FeatureManagementTests.cs

```diff
@@ -928,19 +940,18 @@ public void ValidateCorrectFeatureFlagLoggedIfModifiedOrRemovedDuringRefresh()
             eTag: new ETag("c3c231fd-39a0-4cb6-3237-4614474b92c1" + "f"));
 
             Thread.Sleep(CacheExpirationTime);
-            refresher.LoggerFactory = mockLoggerFactory.Object;
             refresher.TryRefreshAsync().Wait();
             Assert.Equal("AllUsers", config["FeatureManagement:MyFeature:EnabledFor:0:Name"]);
-            Assert.True(TestHelpers.ValidateLog(mockLogger, LoggingConstants.RefreshFeatureFlagChanged + "(key: 'myFeature1', label: '')", LogLevel.Debug));
-            Assert.True(TestHelpers.ValidateLog(mockLogger, LoggingConstants.RefreshFeatureFlagValueUpdated + "'myFeature1'", LogLevel.Information));
+            Assert.Contains(LoggingConstants.RefreshFeatureFlagRead + " Key:'myFeature1' Label:''", verboseInvocation);
```

Comment:
We might benefit from reusing the helper methods like BuildFeatureFlagReadMessage to validate the logged errors. That way, if we change the formatting of error messages, we dont have to update all the tests.

Reply:
Do you mean creating a helper for each kind of error message as well? I agree with using the existing helpers to validate logging tests but not sure what you meant by reusing the feature flag method for logged errors.

Reply:
I meant that we could do something like this:
`Assert.Equal(verboseInvocation, LogHelper.BuildFeatureFlagReadMessage("myFeature1", null))`

Reply:
Ah ok understood, just thought you meant something else as well